src/main/java/net/neoforged/moddevgradle/internal/ModDevPlugin.java

```diff
@@ -259,6 +260,15 @@ public void apply(Project project) {
             minecraftClassesArtifact = createArtifacts.map(task -> project.files(task.getCompiledArtifact()));
         }
 
+        var supplyDevLogin = project.provider(() -> extension.getRuns().stream().anyMatch(model -> model.getDevLogin().get()));
```

Comment:
Wouldn't it be cleaner to map `extension.getRuns()`?

Reply:
Sure

Reply:
FYI: Was kinda pointless in the end so I sort of undid it. I was thinking that the NDOC might have been a provider, but that doesn't seem to be the case.